src/js/components/icons/Spinning.js

```diff
@@ -1,22 +1,25 @@
 // (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
 
-import React, { Component } from 'react';
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
 import CSSClassnames from '../../utils/CSSClassnames';
 
 const CLASS_ROOT = CSSClassnames.SPINNING;
 
 export default class Spinning extends Component {
   render () {
-    var classes = [CLASS_ROOT];
-    if (this.props.small) {
-      classes.push(CLASS_ROOT + "--small");
-    }
-    if (this.props.className) {
-      classes.push(this.props.className);
-    }
+    const { className, small, ...props } = this.props;
+    const classes = classnames(
+      CLASS_ROOT,
+      {
+        [`${CLASS_ROOT}--small`]: small
+      },
+      className
+    );
+
     return (
-      <svg className={classes.join(' ')} viewBox="0 0 48 48" version="1.1"
-        role='img'>
+      <svg className={classes} viewBox="0 0 48 48" version="1.1"
+        role="img" {...props} >
```

Comment:
We should put `{...props}` first to avoid overriding the other properties we pass.